src/Servers/Kestrel/Transport.Quic/test/QuicConnectionListenerTests.cs

```diff
@@ -97,25 +97,5 @@ static void AssertTlsConnectionFeature(IFeatureCollection features, X509Certific
                 Assert.Equal(testCert, tlsFeature.ClientCertificate);
             }
         }
-
-        [ConditionalFact]
-        [MsQuicSupported]
-        // https://github.com/dotnet/runtime/issues/57308, RemoteCertificateValidationCallback should allow us to accept a null cert,
-        // but it doesn't right now.
-        [OSSkipCondition(OperatingSystems.Linux | OperatingSystems.MacOSX)]
-        public async Task ClientCertificate_Required_NotSent_ConnectionAborted()
-        {
-            await using var connectionListener = await QuicTestHelpers.CreateConnectionListenerFactory(LoggerFactory, clientCertificateRequired: true);
-
-            var options = QuicTestHelpers.CreateClientConnectionOptions(connectionListener.EndPoint);
-            using var clientConnection = new QuicConnection(options);
-
-            var qex = await Assert.ThrowsAsync<QuicException>(async () => await clientConnection.ConnectAsync().DefaultTimeout());
-            Assert.Equal("Connection has been shutdown by transport. Error Code: 0x80410100", qex.Message);
-
-            // https://github.com/dotnet/runtime/issues/57246 The accept still completes even though the connection was rejected, but it's already failed.
-            var serverContext = await connectionListener.AcceptAndAddFeatureAsync().DefaultTimeout();
-            await Assert.ThrowsAsync<QuicException>(() => serverContext.ConnectAsync().DefaultTimeout());
```

Comment:
Only these last three lines should be removed, the linked bug was fixed. The rest of the test is still valid and working as expected.